### DISCUSSION TITLE
Running cells with '.venv (Python 3.12.0)' requires the ipykernel package.

### DISCUSSION THREAD

**Discussion by ojrudd** [microsoft/vscode-jupyter#16178] [Questions and Answers]
I have the ipykernel package installed in my venv and this is what I am using as my selected interpreter but no matter what I do it continues to tell me the same message. Any ideas?

**Comment by DonJayamanne** [comment]
Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `trace`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel (use the command `Jupyter: Show Output` to get to the logs).

**Reply by ojrudd** [reply]
<img width="652" alt="Screenshot 2024-10-30 101835" src="https://github.com/user-attachments/assets/74b88e71-e298-4d9c-96e9-3bafba1b5303">


Is this the output panel?

**Reply by DonJayamanne** [reply]
Thank you , Please share the logs as requested.
> Attempt to repro the issue & then copy the output from the Jupyter output panel (use the command Jupyter: Show Output to get to the logs).

**Reply by ojrudd** [reply]
Hi @DonJayamanne sorry I am unsure what logs you mean, where am I meant to run the Jupyter: Show Output command?

**Reply by DonJayamanne** [reply]
Open command palette in vscode, search for that command
